hw/top_englishbreakfast/data/alert_handler_env_pkg__params.sv.tpl

```diff
@@ -0,0 +1,17 @@
+// Copyright lowRISC contributors.
```

Comment:
just a general comment on tpl files that are identical among different tops: would it make sense to either move them into a folder outside the top (e.g. `hw/topgen/data`), or just simlink them to the originals under `hw/top_earlgrey/data`? Otherwise it looks like we're duplicating a lot of information here...

Reply:
I have thought about this, too. But in the end, we want to have the flexibility of different templates. So I decided there was little benefit in moving them to a shared folder. However, your idea of symlinking them sounds exactly like what we need here! We can then create a copy, when it's actually needed. Great idea!